src/DataWrangling/Copernicus/Copernicus.jl

```diff
@@ -20,6 +20,7 @@ import ClimaOcean.DataWrangling:
     latitude_interfaces,
     z_interfaces,
     metadata_filename,
+    metadata_path,
```

Comment:
`metadata_path` is a method of Oceananigans.Fields?

Reply:
it's a method of `ClimaOcean.DataWrangling`

Reply:
Yes. That was my impression. Then how come we import it here from Oceananigans.Fields? Typo?

Reply:
I think you misread the changes. We are importing `metadata_path` from `ClimaOcean.DataWrangling`. From Oceananigans.Fields we are importing `location`
https://github.com/CliMA/ClimaOcean.jl/blob/64b821b0986435474ff486d2d2efa429d641bdc0/src/DataWrangling/Copernicus/Copernicus.jl#L12-L25

Reply:
we shouldn't need to extend the path. why do we need to extend it here?

Reply:
definitely I misread...